lib/environment.ts

```diff
@@ -0,0 +1,24 @@
+import { Env } from '@humanwhocodes/env';
+
```

Comment:
Should add example environment variables to explain what is expected to be exported